src/utils/execution/stepResults.ts

```diff
@@ -31,9 +39,9 @@ export const parseData = (data: any, contentType = ""): string | Buffer => {
         throw new Error("Received malformed JSON payload.");
       }
     }
-    return data;
+    return data as string;
```

Comment:
What type is being inferred here that is forcing a cast?

Reply:
The type of the `data` param was `unknown`, but in reality it's Buffer/JsonValue/unknown. Just updated the types.